tests/robot-tests/tests/admin_and_public/bau/publication_owner.robot

```diff
@@ -0,0 +1,340 @@
+
+Give Analyst1 User1 release access
+    [Tags]  HappyPath
+    # stale element exception if you don't wait until it's enabled
+    user waits until element is enabled  //*[tbody]//tr[1]//td[3]//a
+    user clicks element  //*[tbody]//tr[1]//td[3]//a
+    user waits until element is enabled  css:[name="selectedReleaseId"]
+    user selects from list by label  css:[name="selectedReleaseId"]  ${RELEASE_NAME}
+
+    user waits until element is enabled  css:[name="selectedReleaseRole"]
+    user selects from list by label  css:[name="selectedReleaseRole"]  Approver
+    user clicks button  Add release access
+    
+Sign in as Analyst1 User1 & navigate to publication
+    [Tags]  HappyPath
+    user signs in as analyst1
+    user goes to url  %{ADMIN_URL}
+    user selects theme and topic from admin dashboard  %{TEST_THEME_NAME}  %{TEST_TOPIC_NAME}
+    user waits until page contains accordion section   ${PUBLICATION_NAME}  120
+
+Navigate to release
+    [Tags]  HappyPath
+    user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}  Academic Year 2025/26 (not Live)
+
+Upload subject file
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user waits until page contains element  id:dataFileUploadForm-subjectTitle
+    user enters text into element  id:dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
+    user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}seven_filters.csv
+    user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}seven_filters.meta.csv
+    user clicks button  Upload data files
+    user waits until h2 is visible  Uploaded data files
+    user waits until page contains accordion section   ${SUBJECT_NAME}
+    user opens accordion section   ${SUBJECT_NAME}
+    ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+    user checks headed table body row contains  Status           Complete  ${section}  180
+
+Add meta guidance to ${SUBJECT_NAME} subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document  90
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user enters text into meta guidance data file content editor  ${SUBJECT_NAME}
+    ...  meta guidance content
+    user clicks button  Save guidance
+
+Navigate to 'Footnotes' page
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Add footnote to ${SUBJECT_NAME}
+    [Tags]  HappyPath
+    user waits until page contains link   Create footnote
+    user clicks link  Create footnote
+    user waits until h2 is visible  Create footnote
+    user clicks footnote radio   ${SUBJECT_NAME}   Applies to all data
+    user clicks element  id:footnoteForm-content
+    user enters text into element  id:footnoteForm-content  Footnote 1 ${FOOTNOTE_TEXT_2}
```

Comment:
I wouldn't have the footnote text be half hard-coded and half a variable.

My preference would be to hardcode the footnote text here and delete the variable, since we're not checking elsewhere that the footnote has been successfully created